### ISSUE TITLE
generated zip cannot be extracted on Windows 10

### ISSUE THREAD

**Issue by nahueltaibo** [firmata/firmata-builder#13]
Selectig the options
Serial - 115200
DigitalOutputFirmata 
FirmataScheduler 
with name "Neck"
Created file Neck.zip
Inside the zip, there is a Neck.ino FOLDER, with no Neck.ino sketch

**Comment by soundanalogous** [comment]
Thanks for reporting. I just tried to repro but didn't have an issue. It's interesting however that a folder named "Neck.ino" was generated. I'll look into the server code to see if I can figure out how this error may have occurred.

Which OS and web browser did you use?

Also, did you go through the form top to bottom in a single process or did you check and uncheck some options before finally clicking on the build button?

If you try the same selection again do you get the same error?

**Comment by nahueltaibo** [comment]
I can reproduce it with every order I try.
I  just reproduce it going from top to bottom and just checking DigitalInputFirmata and setting the name as "Name".
The outcome was an empty Name.zip file (no Name.ino folder nor sketch inside)
I asume you are now making changes, thats why there is no folder now. But it is still not working for me.

I'm using Windows 10 and Chrome browser.

Thanks

**Comment by soundanalogous** [comment]
I haven't made any changes. This is very strange. I will test with Win 10 and Chrome.

**Comment by soundanalogous** [comment]
I tested on Win 10 with the same results as you. I also tested on Ubuntu 14.04 with Chrome and that did work. What I suspect the issue is in how Windows extracts zip files because if I copy the zip file I downloaded on Windows over to OS X, I can extract it successfully. I noticed that when attempting to extract the zip in Win 10 it warns me about the file size being too big for the destination so I think Win interprets the size in a weird way. I will need to look into how I can create the zip server side so that it will unzip correctly on all platforms.
